includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -161,7 +153,11 @@ private function filter_attributes( $attributes ) {
 					}
 					break;
 
+				case 'scrolling':
+					break;
```

Comment:
@juanchaur1 I don't think this is right because `scrolling` is allowed on `amp-iframe`:

https://github.com/ampproject/amphtml/blob/0a7e981cf65ef61394fe36f532c2b26a1b653606/extensions/amp-iframe/validator-amp-iframe.protoascii#L56-L58

In any case, it would be better to let the whitelist sanitizer remove it so that a validation error would be reported.